.gitignore

```diff
@@ -32,3 +32,4 @@ repodata/
 !.vscode/tasks.json
 !.vscode/launch.json
 !.vscode/extensions.json
+metron-interface/metron-alerts/TODO
```

Comment:
Should this be here?

Reply:
No. I going to remove it. It was just for my todo file while I was developed this.